Doc/glossary.rst

```diff
@@ -827,10 +827,11 @@ Glossary
       Some named tuples are built-in types (such as the above examples).
       Alternatively, a named tuple can be created from a regular class
       definition that inherits from :class:`tuple` and that defines named
-      fields.  Such a class can be written by hand or it can be created with
-      the factory function :func:`collections.namedtuple`.  The latter
-      technique also adds some extra methods that may not be found in
-      hand-written or built-in named tuples.
+      fields.  Such a class can be written by hand, or it can be created by
+      inheriting :class:`typing.NamedTuple`, or with the factory function
+      :func:`collections.namedtuple`.  The latter technique also adds some
```

Comment:
The last sentence no longer makes sense as "The latter technique" now properly refers to two techniques.
```suggestion
      :func:`collections.namedtuple`.  The latter techniques also add some
```